bin/run.sh

```diff
@@ -40,11 +40,11 @@ set +e
 # Un-skip all the skipped tests
 sed -i 's/skip <|//g' tests/Tests.elm 2> stderr.txt
 STATUS=$?
-cat stderr.txt
+cat stderr.txt 1>&2
 if [ $STATUS -ne 0 ]; then
     jq -n --rawfile m stderr.txt '{version: 3, status: "error", message:$m}' > $OUTPUT_DIR/results.json
-    echo "An error occured while un-skipping the tests."
-    exit 0
+    echo "An error occured while un-skipping the tests." 1>&2
```

Comment:
This is unrelated to fuzz tests, but I realized the other day that some error messages don't appear in the CI, making it hard to debug. I think this should help. It could even help with the flaky tests we sometimes have?